smarts/core/smarts.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         traffic_sim,  # SumoTrafficSimulation
         envision: Optional[EnvisionClient] = None,
         visdom: Optional[VisdomClient] = None,
-        fixed_timestep_sec: float = 0.1,
+        fixed_timestep_sec: Optional[float] = 0.1,
```

Comment:
Why use `Optional` here?

`Optional[...]` is a shorthand notation for `Union[..., None]` .

```suggestion
        fixed_timestep_sec: float = 0.1,
```